JDI.Light/JDI.Light/Elements/Base/UIElement.cs

```diff
@@ -57,6 +61,7 @@ public string Name
         public bool Displayed => Invoker.DoActionWithResult("Is element displayed", () => FindImmediately(() => WebElement.Displayed, false));
         public bool Hidden => Invoker.DoActionWithResult("Is element hidden", () => !IsDisplayedAction(this));
         public Func<UIElement, bool> WaitDisplayedAction => el => WebElement.Displayed;
+        public string PageName;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/42_/jdi-light-csharp/pullRequest?prid=5599881)